app/src/main/report/components/Sidebar/Sidebar.tsx

```diff
@@ -45,19 +45,6 @@ export const mapStateToProps = (state: ReportAppState): Partial<SidebarProps> =>
 
 export const SidebarComponent: React.StatelessComponent<SidebarProps> = (props: SidebarProps) => {
 
-    const changelogLink = props.isReviewer
```

Comment:
At this point, Sidebar doesn't need isReviewer on its props any more since the Changelog tab is now enabled for all users - but I'm going to leave it on as it will be useful for the Run functionality we'll be adding shortly. (Or should we be looking for isRunner perm for that rather than reviewer?..)